scripts/nats-bootstrap/README.md

```diff
@@ -0,0 +1,62 @@
+### NATS auth terminology
+
+- `Accounts` group users and Jetstream support is enabled/disabled at the account level NOTE: pub/sub permissions set at the Account level are not inherited by users of that account.
+- `Signing keys` are associated with accounts and can be 'templated' with pub/sub permissions and have a `role` name set.
+- `Users` are applications connecting to NATS to pub/sub, users can be assigned a `role`G.
+
+## Nats accounts configuration
+
+For a new instance of this sandbox, follow the steps listed in [restore](#restore).
+
+To add accounts/users/change pub sub rights, follow the steps in [bootstrap](#bootstrap).
+
+### restore
+
+To restore previously setup accounts to a new NATS service,
+
+1. copy over [nats-accounts.tar.gz](nats-accounts.tar.gz) into the `nats-box` pod
+
+```
+kubectl cp nats-accounts.tar.gz nats-box-684d684c7c-b48d6:/nats-accounts.tar.gz
+```
+
+2. extract the contents on the tarball
+```
+cd / && tar -xvzf nats-accounts.tar.gz
+```
+
+3. push the accounts to the NATS server
+```
+export NKEYS_PATH=/root/nsc/nkeys && \
+export NSC_HOME=/root/nsc/accounts && \
+nsc push --system-account SYS -u nats://nats:4222 -A
+```
+
+4. Optional - update the confimaps with the accounts
+
+### bootstrap
+
+The Operator account, System Account, Serverservice, Controller accounts
+and users are setup through [nsc-bootstrap.sh](nsc-bootstrap.sh)
+
+To setup these accounts/change credentials/pubsub rights,
+
+1. make sure the nats-box pod is running, then copy over the boostrap script into the nats box pod.
```

Comment:
Does the pod need to be running here? could you bake this into the container-creation step (which would, admittedly, be different for staging use vs. production)?